accessibility/keyboard-accessibility/keyboard-accessibility.md

```diff
@@ -0,0 +1,56 @@
+
+[Enabling Keyboard Accessibility on a Mac](https://dequeuniversity.com/mac/keyboard-access-mac) (Deque University)
+
+Once system-wide keyboard accessibility has been enabled, test the site based on the following considerations. These steps are based on DeQue University’s Basic Testing Routine for Accessibility. Navigation for all of the following tests begins in the URL field of the browser.
+
+
+### Tab Focusability and Tab Order
+The user should be able to navigate to all elements that would naturally receive focus using the `Tab` key to move forward and the `Shift` + `Tab` key to move backwards. As the user tabs through the page, the reading order of the content should be in a logical order.  
+
+Using a positive `tabindex` value to change the tab order should be avoided, as this creates a situation in which the tab order of the focusable elements is not logical to the user and may cause them to need to start from the beginning in order to reach the element they want.
+
+### Keyboard Functionality
+When testing for keyboard functionality, it is important to make sure that all functionality that exists on a web page is available when using only the keyboard. This includes interacting with form elements and any custom widgets that exist on the page.  
+
+A [list of standard keystrokes for common online interactions](https://webaim.org/techniques/keyboard/#testing) can be found on WebAIM’s Keyboard Accessibility guide.
+
+Using native html elements is the most effective way to make a site accessible, as these elements have built in behaviors and patterns that users expect.  If creating a custom widget is necessary, the widget should be developed to allow keyboard users to follow the common pattern of using `Tab`  to enter and exit the widget and then using the arrow keys to navigate within the widget.
+
+### Customer Keyboard Shortcuts and Screen Readers
+Navigation with the keyboard only is not the same as navigating with a keyboard and screen reader. When a screen reader is active, it allows the user to use a unique set of keyboard shortcuts to navigate and interact with their computer and the internet. These shortcuts are not available to users who are not using a screen reader, such as a sighted user who may use keyboard only navigation rather than a mouse due to a motor disability.
+
+Additionally, ensure that any custom keyboard shortcuts do not interfere with the common keyboard shortcuts used by screen readers. There are many screen reader keyboard shortcuts depending on the screen reader in use. Deque University’s [Screen Reader Keyboard Shortcuts and Gestures](https://webaim.org/techniques/keyboard/#testing) is a good resource.
+
+### Visual Focus Indicator
```

Comment:
This section seems like a great place to mention and link to [Safe Focus](https://github.com/sparkbox/safe-focus)